charts/system/intel-gpu-plugin/values.yaml

```diff
@@ -0,0 +1,54 @@
+image:
+  repository: intel/intel-gpu-plugin
+  pullPolicy: IfNotPresent
+  tag: 0.30.0@sha256:e96c9955e5aced52f215f7eff77f679c4c400a16a42b1f2c0e8ed9ba76cae3c4
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+workload:
+  main:
+    podSpec:
+      type: DaemonSet
+      containers:
+        main:
+          args:
+            - -shared-dev-num
+            - "5"
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+
+portal:
+  open:
+    enabled: false
+
+operator:
+  register: true
+
+persistence:
+  devfs:
+    enabled: true
+    type: device
+    mountPath: /dev/dri
+    hostPath: /dev/dri
+    readOnly: true
+  sysfs:
+    enabled: true
+    type: hostPath
```

Comment:
This is probably a "device" as well.
(technically its a hostpath underneath, we just use device to do additional tthings)

Reply:
I tried to google and figure out what was what. Will switch to a device.

Reply:
Its "internal"
https://github.com/truecharts/library-charts/blob/main/library/common/templates/lib/pod/volumes/_device.tpl

(additional checks mostly with some helpful hadling)

Reply:
Got it, already added the commit.